src/util/nodeHelper.ts

```diff
@@ -47,7 +47,7 @@ export default class NodeHelper {
 
     public static routePathHasStop(routePaths: IRoutePath[], nodeId: string) {
         return routePaths.some(routePath =>
-            routePath.routePathLinks.some(routePathLink =>
+            routePath.routePathLinks!.some(routePathLink =>
```

Comment:
Does this need to account for a stop not being in use on the routepathlink?

Reply:
Yes, If I understand you correctly. The routePathLink data is not needed in routePath view, so I don't fetch it. Which means I need to allow it to be empty